src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -226,5 +226,31 @@ public async Task<ActionResult<CreateOrderResponseModel>> CreateOrderAsync([From
             createOrderResponse.Errors = result.Errors.Select(x => new ErrorModel(x.Id, x.Field));
             return BadRequest(createOrderResponse);
         }
+
+        [HttpDelete]
+        [Route("{orderId}")]
+        [Authorize(Policy = PolicyName.CanManageOrders)]
+        public async Task<ActionResult> DeleteOrderAsync(string orderId)
+        {
+            var order = await _orderRepository.GetOrderByIdAsync(orderId);
+
+            if (order is null || order.IsDeleted)
```

Comment:
I don't think the `order.IsDeleted` is really needed here especially after the work @SimonLLawrence is doing with the global query filter. Suppose it could be classed as belt & braces, so up to you if feel it could be removed.

Reply:
As that work isn't in yet, I'll leave it in. Once Simon's stuffs in we could remove it though for sure